HW1/m_1.py

```diff
@@ -0,0 +1,11 @@
+# найдите сумму цифр трехзнаяного числа
+
+print("введите трехзначное число")
+m = int(input())
+c = m // 100
+b = m % 100
+t = m // 10 - c * 10
```

Comment:
Наверное проще тут сделать так:
```suggestion
t = m // 10 % 10
```